modules/ROOT/pages/assigning-permissions.adoc

```diff
@@ -0,0 +1,430 @@
+
+Perform the following steps to manually create the permission set, add the necessary permissions, and assign a user to the permission set:
+
+. Create the AEH_Anypoint permission set:
++
+.. Log in to Salesforce as the System Administrator.
+.. In *Setup*, enter `permission sets` in the Quick Find box and click *Permission Sets*.
+.. Click *Add*.
+.. In the *Label* field, enter `AEH_Anypoint` exactly as it appears.
+.. In the *API Name* field, enter `AEH_Anypoint` exactly as it appears.
+.. Click *Save*
+.. In the *Systems* section, click *System Permissions*.
+.. Click *Edit*.
+. Add the necessary permissions to the permission set:
+.. In the *System* section, enable the following _required_ permissions:
++
+* Apex REST Services
+* API Enabled
+* Assign Topics
+* Author Apex
+* Connect Organization to Environment Hub
+* Create and Customize Dashboards
+* Create and Customize List Views
+* Create and Customize Reports
+*  Create and Set Up Experiences
+* Create Dashboard Folders
+* Create Report Folders
+* Create Topics
+* Customize Application
+* Delete Topics
+* Download AppExchange Packages
+* Edit Events
+* Edit HTML Templates
+* Edit My Dashboards
+* Edit My Reports
+* Edit Tasks
+* Edit Topics
+* Manage Auth. Providers
+* Manage Connected Apps
+* Manage Custom Permissions
+* Manage Dashboards in Public Folders
+* Manage Letterheads
+* Manage Multi-Factor Authentication in User Interface
+* Manage Public Classic Email Templates
+* Manage Public Documents
+* Manage Public List Views
+* Manage Reports in Public Folders
+* Manage Session Permission Set Activations
+* Modify All Data
+* Modify Metadata Through Metadata API Functions
+* Run Reports
+* Transfer Record
+* View All Data
+* View Dashboards in Public Folders
+* View Event Log Files
+* View Login Forensics Events
+* View Real-Time Event Monitoring Data
+* View Reports in Public Folders
+* View Roles and Role Hierarchy
+* View Setup and Configuration
+.. In the *User* section, enable the following _required_ permissions:
++
+* Assign Permission Sets
+* Manage Internal Users
+* Manage IP Addresses
+* Manage Login Access Policies
+* Manage Password Policies
+* Manage Profiles and Permission Sets
+* Manage Roles
+* Manage Sharing
+* Manage Users
+* Reset User Passwords and Unlock Users
+* View All Users
+.. Enable any other permissions that you want to add to the permission set.
+.. Click *Save*.
+. Assign the permission set to a Salesforce user:
++
+.. In *Setup*, enter `Users` in the Quick Find box and click *Permission Sets*, and click *Users*.
+.. In the *Full Name* column, select the name of the user.
+.. In *Permission Set Assignments* section, click *Edit Assignments*.
+.. Select *AEH_Anypoint* from the *Available Permission Sets* list and move the set to the *Enabled Permission Sets* list.
```

Comment:
Can you just say "Move *AEH_Anypoint* from the *Available Permission Sets* list to the *Enabled Permission Sets* list."?

Reply:
This is how it's documented in SF. I think it is ok.